### PR TITLE
Document the ability to use env vars in ldap.toml

### PR DESCRIPTION
**Pull request by svenstaro**
**What this PR does / why we need it**: This was implemented but never documented.

**Which issue(s) this PR fixes**:

This is related to:
https://github.com/grafana/grafana/issues/8832
https://github.com/grafana/grafana/issues/5248
https://github.com/grafana/grafana/pull/20173

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/auth/ldap.md [hunk 1]
```diff
@@ -92,6 +92,14 @@ member_of = "memberOf"
 email =  "email"
 ```
 
+### Using environment variables
+
+You can interpolate variables in the TOML config from environment variables. For instance, you could externalize your `bind_password` that way:
```
Comment (oddlittlebird) [inline docs/sources/auth/ldap.md:97]:
```suggestion
You can interpolate variables in the TOML config from environment variables. For instance, you could externalize your `bind_password` like this:
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

**Review by marefr — APPROVED** [review]
LGTM

**Review by oddlittlebird — APPROVED** [review]

File: docs/sources/auth/ldap.md (hunk 1 above)
Comment (sunkarakamal) [reply docs/sources/auth/ldap.md:97]: Hi oddli
This my docker file looks like 
ARG BASE_IMAGE=alpine:3.12
FROM ${BASE_IMAGE}

ENV LDAP_ADMIN_PASSWORD

and I pass through  Docker run , But it never changed  in ldap config file ?
Is the any documentation How we can execute or sample docker file ?. I have been struggling from 3 days  :(

**Comment by sunkarakamal** [comment]
Finally cracked  to  pass env variables to ldap.toml.Below is my Docker file looks like
ARG BASE_IMAGE=alpine:3.12
FROM ${BASE_IMAGE}

ENV LDAP_ADMIN_PASSWORD=
ENV DOMAIN_NAME=
ENV AWS_ACCESS_KEY_ID=
ENV AWS_SECRET_ACCESS_KEY=
ENV AWS_DEFAULT_REGION=

ARG GRAFANA_TGZ=“grafana-6.5.3.linux-amd64.tar.gz”

And in run.sh file added the below lines for change

“#configuring the Ldap settings for grafana”
bind_password="$LDAP_ADMIN_PASSWORD"
hostname="$DOMAIN_NAME"
sed -i “s/127.0.0.1/$hostname/” /etc/grafana/ldap.toml
sed -i “s/secret/$bind_password/” /etc/grafana/ldap.toml

It worked perfectly .when ever we run docker run or deploy in ecs fargte need to pass the required ENV variables … that mentioned in the Docker file.

**Comment by oddlittlebird** [comment]
Please open a new issue for this or ask in community.grafana.com.

**Pull request merged**